src/main/views/macros/elements/alert/types/alert.ts

```diff
@@ -0,0 +1,7 @@
+export class Alert {
+  content: string;
+
+  constructor(content: string) {
+    this.content = content;
+  }
+}
```

Comment:
I think Typescript lets you simplify this like below, please check- 
` export class Alert {
   constructor(content: string) {}
}`

Reply:
We could one line it but easier for the next dev to edit if we didn't. Why do you feel we should?